tests/foreman/cli/test_repository.py

```diff
@@ -2138,6 +2139,31 @@ def test_positive_sync_sha_repo(self, repo_options):
         sha_repo = Repository.info({'id': sha_repo['id']})
         assert sha_repo['sync']['status'] == 'Success'
 
+    @pytest.mark.tier2
+    @pytest.mark.parametrize(
+        'repo_options',
+        **parametrized([{'content_type': 'yum', 'url': CUSTOM_3RD_PARTY_REPO}]),
+        indirect=True,
+    )
+    def test_positive_sync_third_party_repo(self, repo_options):
+        """Sync 3rd party repos successfully
```

Comment:
since we say "third party" in the tests name and in the new UI test. let's keep it consistent and use "third" instead of 3rd